src/encoded/commands/deploy.py

```diff
@@ -441,119 +294,243 @@ def _wait_and_tag_instances(main_args, run_args, instances_tag_data, instances,
-        print("Spot instance request had been completed, please check to be sure it was fufilled")
-    else:
-        bdm = _get_bdm(main_args)
-        instances = ec2_client.create_instances(
-            ImageId=main_args.image_id,
-            MinCount=run_args['count'],
-            MaxCount=run_args['count'],
-            InstanceType=main_args.instance_type,
-            SecurityGroups=run_args['security_groups'],
-            UserData=run_args['user_data'],
-            BlockDeviceMappings=bdm,
-            InstanceInitiatedShutdownBehavior='terminate',
-            IamInstanceProfile={
-                "Name": run_args['iam_role'],
-            },
-            Placement={
-                'AvailabilityZone': main_args.availability_zone,
-            },
-        )
-        _wait_and_tag_instances(main_args, run_args, instances_tag_data, instances)
-        if 'master_user_data' in run_args and main_args.single_data_master:
-            # ES MASTER instance when deploying elasticsearch data clusters
-            if run_args['master_user_data'] and run_args['count'] > 1 and main_args.elasticsearch == 'yes':
-                instances = ec2_client.create_instances(
-                    ImageId='ami-2133bc59',
-                    MinCount=1,
-                    MaxCount=1,
-                    InstanceType='c5.9xlarge',
-                    SecurityGroups=['ssh-http-https'],
-                    UserData=run_args['master_user_data'],
-                    BlockDeviceMappings=bdm,
-                    InstanceInitiatedShutdownBehavior='terminate',
-                    IamInstanceProfile={
-                        "Name": 'encoded-instance',
-                    },
-                    Placement={
-                        'AvailabilityZone': main_args.availability_zone,
-                    },
-                )
-                _wait_and_tag_instances(main_args, run_args, instances_tag_data, instances, cluster_master=True)
+    bdm = _get_bdm(main_args)
+    instances = ec2_client.create_instances(
+        ImageId=main_args.image_id,
+        MinCount=run_args['count'],
+        MaxCount=run_args['count'],
+        InstanceType=main_args.instance_type,
+        SecurityGroups=run_args['security_groups'],
+        UserData=run_args['user_data'],
+        BlockDeviceMappings=bdm,
+        InstanceInitiatedShutdownBehavior='terminate',
+        IamInstanceProfile={
+            "Name": run_args['iam_role'],
+        },
+        Placement={
+            'AvailabilityZone': main_args.availability_zone,
+        },
+    )
+    _wait_and_tag_instances(main_args, run_args, instances_tag_data, instances)
+    if 'master_user_data' in run_args and main_args.single_data_master:
+        # ES MASTER instance when deploying elasticsearch data clusters
+        if run_args['master_user_data'] and run_args['count'] > 1 and main_args.elasticsearch == 'yes':
```

Comment:
this is the independent master/

Reply:
yes, but we haven't used or tested it in a while.  so I'll see if I can build one.